### DISCUSSION TITLE
Set the number of cores

### DISCUSSION THREAD

**Discussion by filipdvorak7** [nextfoam/baram#48] [Q&A] [answered]
How do I set the number of cores used for the solver? Thank you.

**Comment by jiban** [comment, marked as answer]
The number of cores can be configured in **Parallel** menu.

![image](https://github.com/nextfoam/baram/assets/1674177/39ff01c6-b56c-41d1-95bd-e02c2a97ce9e)

![image](https://github.com/nextfoam/baram/assets/1674177/709661e1-8c12-4567-9be1-f021b38f2013)
